crates/eframe/src/native/file_storage.rs

```diff
@@ -10,15 +10,60 @@ use std::{
 /// [`egui::ViewportBuilder::app_id`] of [`crate::NativeOptions::viewport`]
 /// or the title argument to [`crate::run_native`].
 ///
-/// On native the path is picked using [`directories::ProjectDirs::data_dir`](https://docs.rs/directories/5.0.1/directories/struct.ProjectDirs.html#method.data_dir) which is:
+/// On native, if the `directories` feature is enabled, the path is picked using
+/// [`directories::ProjectDirs::data_dir`](https://docs.rs/directories/5.0.1/directories/struct.ProjectDirs.html#method.data_dir)
+/// which is:
 /// * Linux:   `/home/UserName/.local/share/APP_ID`
 /// * macOS:   `/Users/UserName/Library/Application Support/APP_ID`
-/// * Windows: `C:\Users\UserName\AppData\Roaming\APP_ID`
+/// * Windows: `C:\Users\UserName\AppData\Roaming\APP_ID\data`
```

Comment:
Even directories's own documentation is wrong about this. But the `data` folder is being appended right [here](https://github.com/dirs-dev/directories-rs/blob/da65f064a8bb18d790600433fadeed17a54c793a/src/win.rs#L77).